src/compound/lens/LensStorage.sol

```diff
@@ -38,9 +40,10 @@ abstract contract LensStorage is ILens, Initializable {
     /// CONSTRUCTOR ///
 
     /// @notice Constructs the contract.
-    /// @param _morpho The address of the main Morpho contract.
-    constructor(address _morpho) {
-        morpho = IMorpho(_morpho);
+    /// @param _lensExtension The address of the Lens extension.
+    constructor(address _lensExtension) {
```

Comment:
maybe it is worth adding a validation check on the address ?

Reply:
The validation check is actually performed 2 lines below, by calling a function of the `LensExtension` interface